mantle/platform/machine/openstack/flight.go

```diff
@@ -15,14 +15,17 @@
 package openstack
 
 import (
+	"time"
+
 	"github.com/coreos/pkg/capnslog"
 
 	"github.com/coreos/mantle/platform"
 	"github.com/coreos/mantle/platform/api/openstack"
 )
 
 const (
-	Platform platform.Name = "openstack"
+	Platform            platform.Name = "openstack"
+	provisioningTimeout               = 3 * time.Minute
```

Comment:
I'm interested to see if this is too low a timeout - I suspect we'll probably be adjusting these over time. Also might be worth a CLI switch at some point to be able to override.